### ISSUE TITLE
Is the splice site track necessary, if I want to generalize Saluki to different ORFs for 3'UTRs from MPRAs?

### ISSUE THREAD

**Issue by Jojo-LIU-LIU** [vagarwal87/saluki_paper#7]
Hi, Vikram! I am using Saluki to predict half-lifes of a series of mRNAs. These mRNAs are consisted of fixed 5'UTR, fixed ORF, and different 3'UTRs.  When I use Saluki, following your practice, I do not put the information of 5' splice site into input matrix. These predicted half-lives could be comparable without infor of 5' splice site. Therefor, I can pick out the best 3'UTR which can bring the longest half-life of a mRNA. 
I want to pick out the best 3'UTR which can bring the longest half-life of a mRNA, even though it has different ORFs. So, if I change ORF from one to another (e.g., COL10A1 to ACTB ) and also do not use infor of 5' splice site, could predictions from different ORFs be comparable? Is it okey that splice site track is always absent?   
Looking forward to your reply. Thanks a lot.

**Comment by vagarwal87** [comment]
I think that should be fine. You can randomly pick, say, 20 5'UTR+ORFs, and attach each to your set of 3' UTRs. Then, for a fixed 5'UTR+ORF, you can measure the deviation from the mean effect across all 3' UTRs, and then average that deviation across many 5'UTR+ORF combinations. Then splice sites shouldn't matter much because you're interested in a relative measurement.

**Comment by Jojo-LIU-LIU** [comment]
Thank you very much, Vikram. I finished this analysis based on your suggestions. 
Now, I obtain the final predictions of a series of 3' UTRs with a fixed 5'UTR+ORF by calculating the mean of 50 predicitons. They are all negative values. I have some little questions as following:
1), The larger the value, the longer the half-life, right?  I set the 3'UTR with the largest predicted value of corresponding mRNA as best 3'UTR, am I right? 
2), Could I directly calculate a correlation coefficient between predicted and observed values? Is it necessary to replace the predicted values with deviations from the mean value across all 3' UTRs? How to deal with the observed values?
Looking foward you reply! Thanks!
